### PR TITLE
v1.0.4-SSR-1367 some components collapses on group field

### PR DESCRIPTION
**Pull request by nmanhit**

### CODE REVIEW

File: src/dropdown/index.ts [hunk 1]
```diff
@@ -295,56 +319,60 @@ export class Dropdown extends LitElement {
...
+          role="menu"
+          aria-hidden="${!this._selectorVisible}"
+          ?hidden="${!this._selectorVisible}"
+        >
+          ${this.items.map((item, number) =>
+            this._getItemTemplate(item, number)
+          )}
+        </ul>
+        <div
+          class="kuc-dropdown__error"
```
Comment (TomokoMiyake) [inline src/dropdown/index.ts:367]: Does it will be "kuc-dropdown__group__error"?

File: src/dropdown/index.ts [hunk 2]
```diff
@@ -387,29 +415,39 @@ export class Dropdown extends LitElement {
...
+          min-width: 180px;
+        }
+        .kuc-dropdown__group {
+          border: none;
+          padding: 0px;
+          height: auto;
+          display: inline-block;
+          width: 100%;
+          margin-inline-start: 0px;
+          margin-inline-end: 0px;
```
Comment (TomokoMiyake) [inline src/dropdown/index.ts:427]: Please change the order of this style description under the "kuc-dropdown[hidden]".

Comment (nmanhit) [reply src/dropdown/index.ts:427]: OK

File: src/dropdown/index.ts [hunk 3]
```diff
@@ -352,13 +383,29 @@ export class Dropdown extends LitElement {
     this._updateContainerWidth();
   }
 
+  private _getLabelWith() {
```
Comment (TomokoMiyake) [inline src/dropdown/index.ts:386]: Is this the mistake of `width`?

```suggestion
  private _getLabelWidth() {
```

Comment (nmanhit) [reply src/dropdown/index.ts:386]: ok

Comment (nmanhit) [reply src/dropdown/index.ts:386]: i fixed it, thank you!

**Pull request merged**